CoreScriptsRoot/CoreScripts/PerformanceStatsManagerScript.lua

```diff
@@ -0,0 +1,191 @@
+
+--[[
+		Filename: PerformanceStatsManagerScript.lua
+		Written by: dbanks
+		Description: Handles performance stats gui.
+--]]
+
+--[[ Services ]]--
+local PlayersService = game:GetService("Players")
+local Settings = UserSettings()
+local GameSettings = Settings.GameSettings
+local CoreGuiService = game:GetService('CoreGui')
+
+--[[ Modules ]]--
+local RobloxGuiFolder = CoreGuiService:WaitForChild("RobloxGui")
+local ModulesFolder = RobloxGuiFolder:WaitForChild("Modules")
+local StatsFolder = ModulesFolder:WaitForChild("Stats")
```

Comment:
If they are packaged with the client we do not have to WaitForChild here, in fact it sometimes hides bugs such as missing files.